doc/api/esm.md

```diff
@@ -461,7 +461,11 @@ So the code below, given the package above, will generate a runtime error:
 
 ```js
 // ./another-module.mjs
-import { another } from 'a-package/m.mjs'; // Imports "another" from ./m.mjs.
+
+// Imports "another" from ./m.mjs. Fails because
+// the "package.json" "exports" field
+// does not allow this import.
```

Comment:
```suggestion
// does not provide an export named `m.mjs`.
```

“allow this import” makes me think that the problem is `{ another }`, not `m.mjs`.